omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -1,26 +1,44 @@
 package org.openmrs.module.attachments.rest;
 
-import org.openmrs.Encounter;
-import org.openmrs.Obs;
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.*;
```

Comment:
So can you run an import cleanup shortcut in your IDE to get only the relevant imports?

In Eclipse that'd be CTRL+SHIFT+O, you might want to look that up for IntelliJ.

Reply:
i'm using the same keymap shortcuts of Eclipse, and it's auto managing imports.
using import cleanup set the wildcard import `org.openmrs.*;`